eagleproject/core/blockchain/harvest/transactions.py

```diff
@@ -137,6 +138,27 @@ def explode_log_data(value):
         out.append(int(value[2 + i * 64 : 2 + i * 64 + 64], 16) / 1e18)
     return out
 
+# get rebase log at block number
+def get_rebase_log(block_number):
+    rebase_log = Log.objects.filter(
+        topic_0='0x09516ecf4a8a86e59780a9befc6dee948bc9e60a36e3be68d31ea817ee8d2c80',
```

Comment:
Perhaps, move this to [this file](https://github.com/OriginProtocol/ousd-analytics/blob/master/eagleproject/core/blockchain/sigs.py) as a constant?

Reply:
Yes! done, thanks